stok.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const Hapi = require('hapi');
+
+class Stok {
+  constructor(options) {
+    this.options = options;
```

Comment:
We need to provide the env var parsing and default values before the app can even pass these settings in.